10up/headstartwp
# Add support for robots.txt file

### Description of the Change

- Add rewrite for robots.txt file to read from WordPress.
- Prevent redirection from WordPress to Front-end for robots.txt, to avoid redirection loop.

Before this change, requests to WordPress for `robots.txt` file were being redirected to NextJS front-end, if `Redirect to Headless Site?` setting is enabled in WordPress.

On the front-end side, before this change, requests to NextJS app, who is using the headless framework, for `robots.txt` file would result in 404, because without a `rewrite`, it would not know anything about it.

### How to test the Change
- Visit `http://wordpress.site/robots.txt`.
- Observe that it's not redirecting to `http://nextjs.site/robots.txt`
- Visit `http://nextjs.site/robots.txt`.
- Observe that it's showing the same contents as the one from `http://wordpress.site/robots.txt`

### Checklist:
